### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by AnyaRatusnyak**

### CODE REVIEW

File: src/main/java/TicketService.java [hunk 1]
```diff
@@ -62,8 +61,12 @@ public static void main(String[] args) {
         List<BusTicket> busTickets = fileWork.readFromFile("src/main/resources/tickets.json");
         TicketService.printTicketsInfo(busTickets);
 
-        TicketDataBaseDao ticketDataBaseDao = new TicketDataBaseDaoImpl();
-        UserDataBaseDao userDataBaseDao = new UserDataBaseDaoImpl(ticketDataBaseDao);
+        AnnotationConfigApplicationContext context = new AnnotationConfigApplicationContext(AppConfig.class);
```
Comment (Alexmansar) [inline src/main/java/TicketService.java]: use abstraction here please - ApplicationContext

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@AnyaRatusnyak Good job)) just one comment))

File: src/main/java/TicketService.java (hunk 1 above)
Comment (AnyaRatusnyak) [reply src/main/java/TicketService.java]: Thank you ) Good day !!

**Review by Alexmansar — APPROVED** [review]
@AnyaRatusnyak Good job)) could be merged)

**Pull request merged**